pages/meshes.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <v-layout column>
+    <h1 class="display-4 ma-5">Meshes</h1>
+
+    <p :class="paragraph">
+      In this guide, you will learn fundamental elements to understand and manipulate meshes in OpenGeode.
+      Let's begin by introducing all kind of meshes you can work with in OpenGeode.
+    </p>
+
+    <h2 :class="section">Mesh gallery</h2>
+
+    <p
+      :class="paragraph"
+    >There are several kind of meshes representing points, lines, surfaces and solids.</p>
+
+    <img src="@/assets/mesh_hierarchy.svg" />
+
+    <p :class="paragraph">
+      All these meshes are available in 2D and 3D
+      (excepted solids that are definitely three-dimensional, and graphs that are dimensionless).
```

Comment:
what about vertex set?
```suggestion
      (excepted solids that are definitely three-dimensional, and vertex set and graphs that are dimensionless).
```